File: PA2_Client/src/edu/ucdenver/tournament/Referee.java
package edu.ucdenver.tournament;

public class Referee {
    private String name;

    private Country country;

    public Referee(String name, Country country){
        this.name = name;
        this.country = country;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Country getCountry(){
        return this.country;
    }

    @Override
    public String toString(){
        return String.format("\nReferee Name: %s \nReferee Country: %s \n", this.name, this.country);
    }
}

File: PA2_Server/src/edu/ucdenver/Tournament/Referee.py
from edu.ucdenver.Tournament import Country


class Referee:
    """keeps track of the name and country of a referee"""

    def __init__(self, name: str, country: Country):
        """initializing referee object"""
        self.__name = name
        self.__country = country

    @property
    def name(self):
        """returning the referees name"""
        return self.__name

    @property
    def country(self):
        """returning the referees country"""
        return self.__country


